packages/xrpl/src/models/transactions/AMMDeposit.ts

```diff
@@ -21,6 +21,25 @@ export enum AMMDepositFlags {
   tfTwoAsset = 0x00100000,
   tfOneAssetLPToken = 0x00200000,
   tfLimitLPToken = 0x00400000,
+  tfWithdrawAll = 0x00020000,
+  tfOneAssetWithdrawAll = 0x00040000,
+  tfTwoAssetIfEmpty = 0x00800000,
+  tfWithdrawSubTx = tfLPToken |
+    tfSingleAsset |
```

Comment:
Can I not use existing flags to designate new flags? I get the following lint error ://

`error  Explicit enum value must only be a literal value (string, number, boolean, etc)  `

Reply:
There shouldn't be any masks here - the enum should just be the flags.
I would use https://xrpl.org/docs/references/protocol/transactions/types/ammdeposit/#ammdeposit-flags as your source of truth instead of rippled.

Reply:
It's the dev's job to `|` the flags together, not ours. `rippled` reuses a lot of these combinations, which is why that code has some combinations. They aren't independent flags, though.

Reply:
ok thanks, fixed it